x-pack/test/observability_functional/apps/observability/index.ts

```diff
@@ -20,5 +20,6 @@ export default function ({ loadTestFile }: FtrProviderContext) {
     loadTestFile(require.resolve('./exploratory_view'));
     loadTestFile(require.resolve('./feature_controls'));
     loadTestFile(require.resolve('./alerts/rules_page'));
+    loadTestFile(require.resolve('./alerts/rule_details'));
```

Comment:
@fkanout Since we have rule_details page implemented outside of alerts page `x-pack/plugins/observability/public/pages/rule_details`, I would recommend to move rule_details functional tests outside of alerts folder.

Same comment goes for rules_page tests. What do you think?

Reply:
@mgiota, I see your point and it's a valid one. I think we went with this option as accessing these pages is via the Alerts page.
I'm thinking now of putting the tests under the pages folder-including the Alerts one, which would reflect the what we have in the plugin folder